Tests/DatadogTests/DatadogObjc/ObjcAPITests/DDConfiguration+apiTests.m

```diff
@@ -94,7 +94,6 @@ - (void)testDDConfigurationBuilderAPI {
     [builder enableLogging:YES];
     [builder enableTracing:YES];
     [builder enableRUM:YES];
-    [builder enableCrashReportingUsing:[DDCrashReportingPlugin new]];
```

Comment:
This makes me think that `DatadogCrashReporter.initialize()` is not possible from Objective-C, right? Is this sth we will tackle later instead of now? Does it relate to other modules as well?

Reply:
Ah yes, good catch! We can easily make it accessible from objc, but we have to come up with a different name because `initialize` is reserved. We could do something like this:
```swift
@objc
public final class DatadogCrashReporter: NSObject {

    public static func initialize(in core: DatadogCoreProtocol = defaultDatadogCore)

    @available (*, unavailable) @objc
    public static func enable()
}
```

The `@available (*, unavailable) @objc` will make it available for objc only.

Reply:
Or just:
```swift
@available(swift, obsoleted: 1) @objc(DatadogCrashReporter)
public final class objc_DatadogCrashReporter: NSObject {

    @objc
    public static func enable() {
        DatadogCrashReporter.initialize()
    }
}
```

I think I prefer this option for isolation, WDYT?

Reply:
Oh 😞, `initialize()` conflict is a pity. Given that we don't yet have [proper documentation](https://docs.datadoghq.com/real_user_monitoring/error_tracking/ios/?tab=cocoapods) for crash reporting in Objective-C, it will be harder to discover (and `initialize()` could be called by mistake). Let's go with `enable()` now and we can visit it when aligning public interfaces.

One more thing - it will be more consistent with other APIs and more idiomatic to Objective-C if we call it `DDCrashReporter`, not `DatadogCrashReporter`.

> The @available(swift, obsoleted: 1) @objc will make it available for objc only.

Interesting!